packages/components/src/components/accordion/accordion.tsx

```diff
@@ -71,7 +77,7 @@ export class Accordion {
 
     return (
       <Host>
-        <Heading class="heading">
+        <Heading class={{ heading: true, sticky: this.sticky }}>
```

Comment:
Why do we need a sticky css class?

Reply:
You are right, in an earlier stage I used this to style it sticky but then I changed it to pass "sticky" into the jss.